### PR TITLE
Update ext.tasks example

### PR DESCRIPTION
**Pull request by iDutchy**
## Changes

This will update the example of the ext.tasks usage. Adding the `start` and `cancel` method and removig the `await asyncio.sleep(1800)` for people who didnt read the note.

### CODE REVIEW

**Comment by iDutchy** [comment]
@ChillFish8 first of all, the examples do not show anything to create a task without ext.tasks. the .sleep() alone isnt enough to create a task.

And since discord.py has introduced ext.tasks, why not use it? 

As for the fixes, this adds the task.start() and task.cancel() to the example. As a lot of people just copy without looking, they would miss these. 

My decision to makes this PR was after someone asked me for help. Their task didnt run and the .sleep() was still in the task too.

So basically, why do it manually when the lib provides a built in way to do it? This also makes it easy to cancel the task with an eval for example. `!eval bot.cogs["TopGG"].update_stats.cancel()`
